zerver/lib/actions.py

```diff
@@ -1704,6 +1707,22 @@ def check_send_private_message(sender: UserProfile, client: Client,
 
     return do_send_messages([message])[0]
 
+@has_request_variables
+def check_send_webhook_message(request: HttpRequest, user_profile: UserProfile,
+                               topic: Text, body: Text,
+                               stream: Optional[Text]=REQ(default=None),
+                               custom_topic: Optional[Text]=REQ("topic", default=None)) -> None:
+    if stream is None:
+        assert user_profile.bot_owner is not None
+        check_send_private_message(user_profile, request.client,
+                                   user_profile.bot_owner, body)
+    else:
+        subject = topic
+        if custom_topic is not None:
+            subject = custom_topic
+        check_send_stream_message(user_profile, request.client,
+                                  stream, subject, body)
+
```

Comment:
I'd rather put this in its own file, `zerver/lib/webhooks/__init__.py`, or maybe better `zerver/lib/webhooks/common.py`.  This doesn't need to be in `actions`, since it's intended to only be called in webhooks, and generally the goal is to only add things to that file that need to be there (it's huge enough as it is).